incubator/java-spring-boot2/image/project/appsody-boot2-pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>dev.appsody</groupId>
   <artifactId>spring-boot2-stack</artifactId>
-  <version>0.3.5</version>
+  <version>0.3.6</version>
```

Comment:
Can we update the version to 0.3.7?

Reply:
I'll update to the most recent version before this is merged.

Reply:
There is another PR which adds maven wrapper to this stack and upgrades the version. We can merge both of these together and release in one go. If that sounds reasonable, please remove version changes from this PR.